src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -21,36 +47,421 @@ const ActorInit En_Bji_01_InitVars = {
     (ActorFunc)EnBji01_Update,
     (ActorFunc)EnBji01_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CCDE0.asm")
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_HIT0,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
+    { 18, 64, 0, { 0, 0, 0 } },
+};
+
+/* Animations struct */
+static struct_80B8E1A8 D_809CDC7C[4] = {
```

Comment:
There's no need to put the length of the array explicitly. We usually don't do this.
```suggestion
static struct_80B8E1A8 D_809CDC7C[] = {
```